ntia_conformance_checker/sbom_checker.py

```diff
@@ -78,23 +80,27 @@ def get_components_without_names(self):
         """Retrieve SPDX ID of components without names."""
         components_without_names = []
         for package in self.doc.packages:
-            if package.name is None:
+            if package.name is None or package.name == "":
                 components_without_names.append(package.spdx_id)
         return components_without_names
 
     def get_components_without_versions(self):
         """Retrieve SPDX ID of components without names."""
         components_without_versions = []
         for package in self.doc.packages:
-            if package.version is None:
+            if package.version is None or package.version == "":
                 components_without_versions.append(package.name)
         return components_without_versions
 
     def get_components_without_suppliers(self):
         """Retrieve name of components without suppliers."""
         components_without_suppliers = []
         for package in self.doc.packages:
-            if package.supplier is None or "NOASSERTION" in package.supplier.name:
+            if (
+                package.supplier is None
+                or "NOASSERTION" in package.supplier.name
```

Comment:
I guess this would be `if not package.suplier or "NOASSERTION" in package.supplier.name`

Reply:
Right again. Will fix!